astroquery/nasa_exoplanet_archive/tests/test_nasa_exoplanet_archive_remote.py

```diff
@@ -121,17 +107,3 @@ def test_hd209458b_exoplanet_archive_coords():
     sep = params['sky_coord'].separation(simbad_coords)
 
     assert abs(sep) < 5 * u.arcsec
-
-
-@remote_data
-def test_exoplanet_archive_table_all_columns():
```

Comment:
While it makes sense to remove these now-obsolete tests, we need something to replace them.  Could you add planet-specific tests here?

Reply:
planet-specific tests are already present. Star-specific tests need to be added up. I'm working on it right now.